content/main_tester.sh

```diff
@@ -152,6 +149,9 @@ error_code_to_string() {
 # Iterate over all testcase IDs passed as command line arguments
 for TESTCASE_ID in "$@";
   do
+    # Add executable permission
+    chmod +x ${PROB_FDR}/${PROB_CODE}/test_script
```

Comment:
Move it outside the loop, it suffices to do the chmod once

Reply:
Oh boi. I didn't realize I was still in loop. Doing